components/brave_extension/extension/brave_extension/manifest.json

```diff
@@ -54,8 +59,8 @@
       "all_frames": true
     }
   ],
-  "permissions": [ "contentSettings", "settingsPrivate", "management", "tabs", "storage", "webNavigation", "contextMenus", "cookies", "*://*/*", "chrome://favicon/*" ],
-  "content_security_policy": "default-src 'self'; font-src 'self' data:; script-src 'self'; style-src 'unsafe-inline'; img-src 'self' data: chrome://favicon/;",
+  "permissions": [ "contentSettings", "settingsPrivate", "management", "tabs", "history", "storage", "webNavigation", "contextMenus", "cookies", "*://*/*", "chrome://favicon/*", "https://pcdn.brave.software/*", "https://brave-today-cdn.brave.software/*" ],
+  "content_security_policy": "default-src 'self' https://pcdn.brave.software https://brave-today-cdn.brave.software; font-src 'self' data:; script-src 'self'; style-src 'unsafe-inline'; img-src 'self' data: chrome://favicon/ https://pcdn.brave.software; connect-src https://pcdn.brave.software https://brave-today-cdn.brave.software;",
```

Comment:
are we going to use brave.software instead of brave.com for this service? if so we might need to do key pinning for it and also add it to the network audit whitelist if it's not there already

Reply:
Actually I think `brave-today-cdn.brave.software` will also change to `brave-today-cdn.brave.com` before this is merged.

Reply:
Still waiting for this to be deployed. It's coming soon as far as I can tell.

Reply:
Just to recap, once the files have been uploaded to the right places, we can:
- put `pcdn.brave.com` and `brave-today-cdn.brave.com` in `connect-src` and `img-src`
- leave `default-src` set to `'self'`

Reply:
@fmarier I've put the staging URLS in `connect-src` and removed any additions to `img-src` and `permissions`. PTAL.